src/components/formSections/components/TermsListField.js

```diff
@@ -259,28 +328,46 @@ export default class TermsListField extends React.Component {
     let termNoteError = false;
 
     const termsList = terms.map((term, i) => {
-      const errorMessage = this.validateNoteField(value, term.value);
+      const errorMessage = this.validateFields(value, term.value);
       termNoteError = errorMessage ? true : termNoteError;
 
       return (
-        <React.Fragment key={term.value}>
+        <Card
+          data-test-term
+          headerStart={
+            <strong data-test-header={term.name}>
+              <FormattedMessage id="ui-licenses.term" />
+              &nbsp;
+              {i + 1}
+            </strong>
+          }
+          headerEnd={
+            this.renderTermDelete(term, i)
+          }
+          key={i}
+        >
           <Row>
-            <Col xs={5}>
+            <Col xs={12}>
               {this.renderTermName(term, i)}
             </Col>
+          </Row>
+          <Row>
             <Col xs={6}>
```

Comment:
How do these sections look on a small screen? Are they squished? Is there enough room for them to be usable? 

If they do appear squished and unusable, it might make sense to define them as `<Col xs={12} md={6}>` so that they're full width on small screens but side-by-side on medium and larger screens.